watchtower/wtclient/client.go

```diff
@@ -1320,9 +1315,13 @@ func (c *TowerClient) RegisteredTowers(opts ...wtdb.ClientSessionListOption) (
 	if err != nil {
 		return nil, err
 	}
-	clientSessions, err := c.cfg.DB.ListClientSessions(
-		nil, c.genSessionFilter(false), opts...,
-	)
+
+	options := []wtdb.ClientSessionListOption{
+		wtdb.WithPreEvalFilterFn(c.genSessionFilter(false)),
+	}
+	options = append(options, opts...)
```

Comment:
I believe this could be simplified as:
```
opts = append(opts, wtdb.WithPreEvalFilterFn(c.genSessionFilter(false)))
```

see: https://go.dev/play/p/U7hnG50edlY

Reply:
added 🤓